client/src/components/payment/fee-summary.vue

```diff
@@ -8,8 +8,8 @@
 
     <v-slide-y-transition group tag="ul" class="fee-list" v-show="!fetchError">
       <template
-        v-show="(totalFilingFees > 0 && lineItem.filingFees) || (totalFilingFees == 0)"
```

Comment:
`totalFilingFees` does not exist. I assume this is supposed to be `totalFees` (getter).